packages/teleport/src/Login/Login.test.tsx

```diff
@@ -35,80 +35,26 @@ test('basic rendering', () => {
   expect(getByText(/sign into teleport/i)).toBeInTheDocument();
 });
 
-test('login with username and password', async () => {
+test('login with redirect', async () => {
   jest.spyOn(auth, 'login').mockResolvedValue(null);
 
   const { getByPlaceholderText, getByText } = render(<Login />);
 
-  // test validation errors
-  fireEvent.click(getByText(/login/i));
-  expect(auth.login).not.toHaveBeenCalled();
-
   // fill form
   const username = getByPlaceholderText(/username/i);
   const password = getByPlaceholderText(/password/i);
   fireEvent.change(username, { target: { value: 'username' } });
   fireEvent.change(password, { target: { value: '123' } });
 
-  // test login pathways
-  await waitFor(() => fireEvent.click(getByText(/login/i)));
+  // test login and redirect
+  await waitFor(() => fireEvent.click(getByText('Sign In')));
   expect(auth.login).toHaveBeenCalledWith('username', '123', '');
   expect(history.push).toHaveBeenCalledWith('http://localhost/web', true);
 });
 
-test('login with password and otp', async () => {
```

Comment:
Hmm, I don't get why we're removing these tests, aren't those login methods still supported?

Reply:
there were similar tests found in [FormLogin.tsx](https://github.com/gravitational/webapps/blob/master/packages/teleport/src/components/FormLogin/FormLogin.test.tsx). I didn't think it was necessary to repeat the test. For this test file though, I kept one test to test the `redirect` path which is the same for login user + mfa combos.